constructors.js

```diff
@@ -124,8 +138,20 @@ console.assert(dontTellNobody.squeal() === mySecret, 'Tell a secret when you `sq
 // the secret data.
 
 // -- ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ - Your Answer - ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ -- //
+function Key() {
+  function GiveMeAKey() {
+    return Key;
+  }
+}
 
-
+function Safe(data1, Key) {
```

Comment:
Capitalized variable names should only be used for constructors/types not arguments or regular variables.